### PR TITLE
fixes #6725

### LINKED ISSUE #6725: /config/plugins.js doesn't merge into local plugins
**Describe the bug**
Whether or not it makes sense to override local plugin `config/settings.js` with the general `/config/plugins.js`, it doesn't work. When I offloaded my plugin to be an installed package, it does work and merges in the values I've set in `/config/plugin.js`

**System**

- Node.js version: 12.16.0
- NPM version:
- Strapi version: 3.0.0
- Database: Mongo
- Operating system: Macos

**Additional context**
I'm developing a local plugin in the `/plugins` folder. I want it to have default settings, and I want to be able to set override values in the global `/config` folder

### PR DESCRIPTION
**Pull request by tudorilisoi**
Signed-off-by: Tudor Ilisoi <tudorilisoi@yahoo.com>

#### Description of what you did:

merged local plugins config with `config/plugins.js`

Fix https://github.com/strapi/strapi/pull/6769

### CODE REVIEW

File: packages/strapi/lib/core/load-plugins.js [hunk 1]
```diff
@@ -37,8 +37,11 @@ const loadLocalPlugins = async ({ dir }) => {
     loadFiles(pluginsDir, '{*/!(config)/*.*(js|json),*/package.json}'),
     loadConfig(pluginsDir, '*/config/**/*.+(js|json)'),
   ]);
-
-  return _.merge(files, configs);
+  const userConfigs = Object.keys(files).reduce((acc, plugin) => {
+    acc[plugin] = config.get(['plugins', plugin], {});
```
Comment (alexandrebodin) [inline packages/strapi/lib/core/load-plugins.js:41]: this is not gonig to be merged in the right place  like that.

You need to put the config in a config property here:
```js
acc[plugin] = { config: config.get() }
```

Comment (tudorilisoi) [reply packages/strapi/lib/core/load-plugins.js:41]: Done.

**Comment by alexandrebodin** [comment]
Sorry @tudorilisoi I can't see any change

**Comment by tudorilisoi** [comment]
@alexandrebodin  now it should be fine

**Pull request merged**